pkg/cmos/cmos.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmos
+
+import (
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+func Read(reg uint64, data memio.UintN) error {
```

Comment:
since this function's parameters already depend on `memio`, why not making also `reg` a `memio.Uint8`?